lua/ggu.lua

```diff
@@ -1,5 +1,9 @@
 return function (net_conf, gk_conf, lcore)
 
+	-- XXX Sample parameters, need to be tested for better performance.
+	local mailbox_max_entries = 128
```

Comment:
Reading https://doc.dpdk.org/api/rte__mempool_8h.html#a503f2f889043a48ca9995878846db2fd , I learned that the optimum size (in terms of memory usage) for a mempool is when n is a power of two minus one: n = (2^q - 1). So let's (1) rename `mailbox_max_entries` to `mailbox_max_entries_exp`, (2) replace `128` with `7`, and add the following comment just before the variable `mailbox_max_entries_exp`:
```
--- For optimum memory usage, the maximum number of elements is defined as (2^mailbox_max_entries_exp - 1).
```
Break comment as needed.

Reply:
Thinking this issue further, the parameter `ele_count` of `init_mailbox()` should be `ele_count_exp` because `rte_ring_create()` asks for a power of 2, and `rte_mempool_create()` would be best served with that power of 2 minus 1.